Makefile

```diff
@@ -111,3 +111,7 @@ dcshell:
 dctest: endtoendtest
 	# launch all studio's dependent services using docker-compose, and then run the tests
 	echo "Finished running  make test -e DJANGO_SETTINGS_MODULE=contentcuration.test_settings"
+
+dcservicesup:
+	# launch all studio's dependent services using docker-compose
+	docker-compose -f docker-compose.yml -f docker-compose.alt.yml up minio postgres redis celery-worker
```

Comment:
IMO, I think we should remove `celery-worker` from this. I think to avoid confusion, that might just be a good idea to keep Python code running on the host. That means we'll need another documentation entry about running `make prodceleryworkers` on the host in the python virtual environment.

Reply:
Agree, people were confused about this during the setup hack session.